### PR TITLE
Fix path with scheme normalization

### PR DESCRIPTION
**Pull request by mvorisek**
scheme grammar: https://www.rfc-editor.org/rfc/rfc3986#page-17

### CODE REVIEW

**Comment by mvorisek** [comment]
PR is done, CI failures are unrelated.

File: src/File/FileHelper.php [hunk 1]
```diff
@@ -43,7 +44,7 @@ public function absolutizePath(string $path): string
 				return $path;
 			}
 		}
-		if (str_starts_with($path, 'phar://')) {
+		if (str_contains($path, '://')) {
```
Comment (ondrejmirtes) [inline src/File/FileHelper.php]: AFAIK this change isn't covered by tests

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: src/File/FileHelper.php (hunk 1 above)
Comment (mvorisek) [reply src/File/FileHelper.php:47]: test added

Comment (mvorisek) [reply src/File/FileHelper.php]: done

**Comment by mvorisek** [comment]
Can this smallfix be merged?

**Comment by ondrejmirtes** [comment]
Thank you.

**Pull request merged**